alluxio-ui/worker/src/containers/pages/BlockInfo/BlockInfo.tsx

```diff
@@ -64,14 +61,13 @@ class BlockInfo extends React.Component<AllProps, IBlockInfoState> {
   }
 
   public componentDidUpdate(prevProps: AllProps) {
-    const {refreshValue, location: {search}} = this.props;
-    const {refreshValue: prevRefreshValue, location: {search: prevSearch}} = prevProps;
+    const {refresh, location: {search}} = this.props;
+    const {refresh: prevRefresh, location: {search: prevSearch}} = prevProps;
     if (search !== prevSearch) {
       const {path, offset, limit, end} = parseQuerystring(search);
       this.setState({path, offset, limit, end});
       this.fetchData(path, offset, limit, end);
-    }
-    if (refreshValue !== prevRefreshValue) {
+    } else if (refresh !== prevRefresh) {
```

Comment:
With this logic, we can only update one `search` or `refresh`, is this intended?

Reply:
Yes, I figure why sent two identical requests, the refresh one will request the same payload it had before, so whenever the search string changes and the refresh is triggered we will only send one.